data/transport missions.txt

```diff
@@ -3469,3 +3469,104 @@ mission "Inheritance Redirection"
 				decline
 	on complete
 		dialog "There is no inheritance for Mr. Pappa waiting for you at the bank, but there are many other victims that paid some credits while expecting to earn a larger amount of credits in return. Some make a big fuss about it, until the alerted authorities start fining them for attempted inheritance fraud. You leave the bank in silence."
+
+
+
+mission "Hauler VI cargo prototype 1"
+	name "Escort Hauler prototype to formal dinner"
+	description "Escort the Testament to <destination> to deliver sushi for a formal dinner."
+	source "Deep"
```

Comment:
I suggest using `near "Hadar" 5 6` (minimum of 5 jumps from and to Hadar/Skymoot including the given system, maximum of 6 jumps) instead of limiting it to Kornephoros/Deep.